src/test/java/org/ays/payload/UserCredentials.java

```diff
@@ -10,4 +12,25 @@ public class UserCredentials {
     private String username;
     private String password;
 
+    public static UserCredentials generateCreate() {
+        User user = User.generateUserPayload();
+        Response response = InstitutionEndpoints.createAUser(user);
+        if (response.getStatusCode() == 200) {
+            return response.then()
+                    .extract().jsonPath().getObject("response", UserCredentials.class);
+        } else {
+            throw new RuntimeException("User creation failed with status code: " + response.getStatusCode());
+        }
+    }
```

Comment:
Bu fonksiyonu buradan kaldırıp `InstitutionEndpoints`'in içerisine taşıyabiliriz. Burası `UserCredentials` üretmiyor, Back-End servisine istek atarak yeniden bir User oluşturuyor ve üzerine Login oluyor aslında.